randovania/games/dread/json_data/Elun.txt

```diff
@@ -581,9 +577,7 @@ Extra - asset_id: collision_camera_008
   * Extra - actor_layer: breakables
   * Extra - tile_types: ('POWERBEAM',)
   > Door to Gyroscope Room
-      Any of the following:
-          Ballspark
-          Movement (Beginner) and Lay Cross Comb
+      Ballspark or Lay Cross Comb
```

Comment:
This should remain beginner movement, especially since someone probably wouldnt even think to do it, and jumping out of the fan is no small feat

Reply:
You told me to add this into it.